### PR TITLE
Add lines in StopPoint

### PR DESCRIPTION
**Pull request by kadhikari**
Ticket: https://jira.kisio.org/browse/NAVP-1630

### CODE REVIEW

**Review by benoit-bst — COMMENTED** [review]
Add lines in StopPoint ? not StopArea

**Comment by xlqian** [comment]
it seems you need to add line into stop_area, too

**Comment by kadhikari** [comment]
Sorry for the mistake. I already added lines in StopArea in https://github.com/CanalTP/navitia-proto/pull/147

**Review by benoit-bst — APPROVED** [review]

**Review by pbench — APPROVED** [review]

File: type.proto [hunk 1]
```diff
@@ -271,7 +271,7 @@ message StopPoint {
     repeated PhysicalMode physical_modes = 18;
     optional FareZone fare_zone = 19;   
     repeated EquipmentDetails equipment_details = 20;
-
+    repeated Line lines = 22;
```
Comment (nonifier) [inline type.proto:274]: There is already a number `22` for   `repeated string impact_uris = 22;`
According to protobuff doc ` each field in the message definition has a unique number. `
```suggestion
    repeated Line lines = 23;
```

**Review by nonifier — CHANGES_REQUESTED** [review]

**Review by xlqian — APPROVED** [review]

**Review by nonifier — APPROVED** [review]
:+1:

File: type.proto (hunk 1 above)
Comment (kadhikari) [reply type.proto]: Thanks :+1: Done

**Pull request merged**